### DISCUSSION TITLE
fuse: unknown option `cache.files=auto-full'

### DISCUSSION THREAD

**Discussion by delboy2028** [trapexit/mergerfs#963] [General]
unable to loads mergerfs.. get the following error on failed to start.
-- The job identifier is 1170.
Sep 19 15:41:47 s10925529 mergerfs[1905]: fuse: unknown option `cache.files=auto-full'
Sep 19 15:41:47 s10925529 systemd[1]: gmedia.service: Control process exited, code=exited, status=1/FAILURE
-- Subject: Unit process exited
-- Defined-By: systemd
-- Support: https://www.debian.org/support
--

here is the service
[Unit]
Description=gmedia mergerfs mount
Requires=rclone.service
After=rclone.service

[Service]
Type=forking
ExecStart=/usr/bin/mergerfs /local:/GD=NC /gmedia -o rw,use_ino,allow_other,func.getattr=newest,category.action=all,category.create=ff,cache.files=auto-full,dropcacheonclose=true
KillMode=process
Restart=on-failure

[Install]
WantedBy=multi-user.target

**Comment by trapexit** [comment]
Provide information as described in the docs otherwise I can't help.

https://github.com/trapexit/mergerfs#information-to-include-in-bug-reports

**Comment by delboy2028** [comment]
mergerfs version: 2.24.2
FUSE library version: 2.9.8-mergerfs
fusermount version: 2.9.9
using FUSE kernel interface version 7.19


Linux s10925529 4.19.0-17-amd64 #1 SMP Debian 4.19.194-3 (2021-07-18) x86_64 GNU/Linux

Linux s10925529 4.19.0-17-amd64 #1 SMP Debian 4.19.194-3 (2021-07-18) x86_64 GNU/Linux
delboy2028@s10925529:~$ df -h
Filesystem      Size  Used Avail Use% Mounted on
udev            7.7G     0  7.7G   0% /dev
tmpfs           1.6G  8.8M  1.6G   1% /run
/dev/md127       29T  7.5G   28T   1% /
tmpfs           7.7G   12K  7.7G   1% /dev/shm
tmpfs           5.0M     0  5.0M   0% /run/lock
tmpfs           7.7G     0  7.7G   0% /sys/fs/cgroup
/dev/md2       1008M   54M  904M   6% /boot
/dev/md126      3.9G   17M  3.7G   1% /tmp
gcrypt:         1.0P     0  1.0P   0% /GD
tmpfs           1.6G     0  1.6G   0% /run/user/1000


still figuring out how to do the other bits uwant..

but its a fresh install, install mergerfs and fuse and then i create the service above. i enable it which works but starting does not.

**Comment by trapexit** [comment]
> mergerfs version: 2.24.2

As the docs say: 

> Please make sure you are using the latest release or have tried it in comparison. Old versions, which are often included in distros like Debian and Ubuntu, are not ever going to be updated and your bug may have been addressed already.

2.24.2 was released in early 2018. Please use a current version of the software.
